mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
You're not clearing the fragment first, so you're inevitably going to end up with garbage later on. Also, the live DOM doesn't do well with shared references or circular ones, so this will break down in scenarios like this:

```js
// First tree
m("div", m.trust("<button><div></div></button>"))

// Second tree
m("div", m("button", m.trust("<div></div>")))
```

Either use `frag.cloneNode()` and clear the old fragment or create a new fragment each time (the latter is better, despite the allocation - this isn't a frequently traveled path).